_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -39,15 +40,20 @@ Each test target's `autocomplete` [attribute value][] is a [space separated][] l
 
 ## Expectation 2
 
-Each test target's `autocomplete` [attribute value][] has a [correct autocomplete field][] that is [appropriate][appropriate field for the form control] for that test target.
+Each test target's `autocomplete` [attribute value][] has a [correct autocomplete field][] that is [appropriate][appropriate field for the form control] for that its control group. It means that, for example, an `input` element with `autocomplete="new-password"` must have the `type` attribute with one of the following values:
+
+- `hidden` (in this case the test target is inapplicable),
+- `text`,
+- `search`,
+- `password`.
```

Comment:
I think that the addition is more of a note than really part of the expectation. Expectations are not the best place to add example…
I think that could be moved to the background, maybe with a slightly lengthier explanation (not just an example).

Reply:
The suggestion was to explain what appropriate means without using the glossary. I'll have a think.